[stftpitchshift] Update to 1.4.1

jurihock opened the pull request:
**Describe the pull request**

- #### What does your PR fix?
  update stftpitchshift to v1.4.1

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all, yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

jurihock committed:
Add stftpitchshift port

jurihock committed:
Fix shared library build on Windows

jurihock committed:
Run vcpkg x-add-version stftpitchshift

jurihock committed:
Update version database

jurihock committed:
Just increase the version counter to hopefully fix the vcpkg problem I simply don't understand... 😡

jurihock committed:
Now update both sha and version...

jurihock committed:
Update version database

BillyONeal committed:
Merge remote-tracking branch 'origin/master' into HEAD

BillyONeal committed:
Fix version database.

BillyONeal committed:
Add quotes and modernize cmake helpers.

jurihock committed:
Merge branch 'microsoft:master' into master

jurihock committed:
Update stftpitchshift to v.1.4.1

jurihock committed:
Update version database like suggested by bot

Cheney-W commented on ports/stftpitchshift/portfile.cmake:
```diff
@@ -14,6 +14,17 @@ vcpkg_cmake_configure(
+
+file(
+  INSTALL "${SOURCE_PATH}/LICENSE"
+  DESTINATION "${CURRENT_PACKAGES_DIR}/share/stftpitchshift"
```
```suggestion
  DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}"
```

jurihock committed:
Update ports/stftpitchshift/portfile.cmake

Funny suggestion, to use some kind of hidden ${PORT} variable just in one place and still the port name string otherwise...

Co-authored-by: Cheney Wang <38240633+Cheney-W@users.noreply.github.com>

jurihock commented:
@Cheney-W My original commit 1513d67 was successful, but after your suggestion 8c82315 there is a build error. Please troubleshoot...

Cheney-W commented:
• .\vcpkg.exe x-add-version {port name} --overwrite-version
	• git add .
	• git commit -m "update json"
	• git push

jurihock committed:
Update version database

jurihock commented:
Yeah, the overhead of manual vcpkg internal fixes (like "update version database" etc.) for just a single commit in the pull request is ridiculous! Instead of spending own free time efficiently to work on open source projects and do something useful...

jurihock committed:
Consequently use ${PORT} everywhere...

and not just in one place like suggested by Cheney-W

jurihock committed:
Update stftpitchshift.json

Cheney-W commented on ports/stftpitchshift/portfile.cmake:
```diff
@@ -14,6 +14,17 @@ vcpkg_cmake_configure(
 
-file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/stftpitchshift" RENAME copyright)
+vcpkg_cmake_config_fixup(
+  PACKAGE_NAME "${PORT}"
```
Thie sentence could be removed, PACKAGE_NAME defaults to ${PORT}.

jurihock committed:
Cheney-W: Thie sentence could be removed, PACKAGE_NAME defaults to ${PORT}.

jurihock committed:
Update stftpitchshift.json

jurihock commented:
So what else?

Cheney-W commented:
Thanks for your prompt reply, now just waiting for merge.

BillyONeal commented:
Thanks for the update!
